openwisp_ipam/admin.py

```diff
@@ -145,6 +161,11 @@ def import_view(self, request):
                 return redirect('/admin/{0}/subnet'.format(self.app_label))
         return render(request, form_template, context)
 
+    def get_csv_organization(self, request):
+        data = Subnet._get_csv_reader(self, deepcopy(request.FILES['csvfile']))
+        org = Organization.objects.get(name=list(data)[2][0].strip())
```

Comment:
Gagan is right, we need to use the organization slug here.
Looking at the code we found similar issues: #99 and #101.

Reply:
So should I leave it here, and take care of this separately in #99?

Reply:
@purhan sorry I forgot to reply! Definitely change this one here, since we're adding it, there's no reason to keep adding it in the wrong way :blush: 

Please could you ping me when ready so I don't forget? Thanks!